test/lightning_client_mock.go

```diff
@@ -119,3 +120,9 @@ func (h *mockLightningClient) AddInvoice(ctx context.Context,
 
 	return hash, payReqString, nil
 }
+
+func (h *mockLightningClient) ListTransactions(
+	ctx context.Context) ([]*wire.MsgTx, error) {
+
+	return nil, nil
```

Comment:
Would be nice if we could control what the mock returns by adding a variable to `test/lnd_services_mock.go` and returning it here.

Reply:
Done, also included in this list the transactions by `SendOutputs` and `PublishTransaction`.